pkg/qdr/qdr.go

```diff
@@ -432,23 +432,25 @@ type LogConfig struct {
-	Http             bool   `json:"http,omitempty" yaml:"http,omitempty"`
-	Cost             int32  `json:"cost,omitempty" yaml:"cost,omitempty"`
-	SslProfile       string `json:"sslProfile,omitempty" yaml:"ssl-profile,omitempty"`
-	SaslMechanisms   string `json:"saslMechanisms,omitempty" yaml:"sasl-mechanisms,omitempty"`
-	AuthenticatePeer bool   `json:"authenticatePeer,omitempty" yaml:"authenticate-peer,omitempty"`
-	LinkCapacity     int32  `json:"linkCapacity,omitempty" yaml:"link-capacity,omitempty"`
-	HttpRootDir      string `json:"httpRootDir,omitempty" yaml:"http-rootdir,omitempty"`
-	Websockets       bool   `json:"websockets,omitempty" yaml:"web-sockets,omitempty"`
-	Healthz          bool   `json:"healthz,omitempty" yaml:"healthz,omitempty"`
-	Metrics          bool   `json:"metrics,omitempty" yaml:"metrics,omitempty"`
-	MaxFrameSize     int    `json:"maxFrameSize,omitempty" yaml:"max-frame-size,omitempty"`
-	MaxSessionFrames int    `json:"maxSessionFrames,omitempty" yaml:"max-session-frames,omitempty"`
+	Name                           string `json:"name,omitempty" yaml:"name,omitempty"`
+	Role                           Role   `json:"role,omitempty" yaml:"role,omitempty"`
+	Host                           string `json:"host,omitempty" yaml:"host,omitempty"`
+	Port                           int32  `json:"port" yaml:"port,omitempty"`
+	RouteContainer                 bool   `json:"routeContainer,omitempty" yaml:"route-container,omitempty"`
+	Http                           bool   `json:"http,omitempty" yaml:"http,omitempty"`
+	Cost                           int32  `json:"cost,omitempty" yaml:"cost,omitempty"`
+	SslProfile                     string `json:"sslProfile,omitempty" yaml:"ssl-profile,omitempty"`
+	SaslMechanisms                 string `json:"saslMechanisms,omitempty" yaml:"sasl-mechanisms,omitempty"`
+	AuthenticatePeer               bool   `json:"authenticatePeer,omitempty" yaml:"authenticate-peer,omitempty"`
+	LinkCapacity                   int32  `json:"linkCapacity,omitempty" yaml:"link-capacity,omitempty"`
+	HttpRootDir                    string `json:"httpRootDir,omitempty" yaml:"http-rootdir,omitempty"`
+	Websockets                     bool   `json:"websockets,omitempty" yaml:"web-sockets,omitempty"`
+	Healthz                        bool   `json:"healthz,omitempty" yaml:"healthz,omitempty"`
+	Metrics                        bool   `json:"metrics,omitempty" yaml:"metrics,omitempty"`
+	MaxFrameSize                   int    `json:"maxFrameSize,omitempty" yaml:"max-frame-size,omitempty"`
+	MaxSessionFrames               int    `json:"maxSessionFrames,omitempty" yaml:"max-session-frames,omitempty"`
+	IdleTimeoutSeconds             int    `json:"idleTimeoutSeconds,omitempty" yaml:"idle-timeout-seconds",omitempty"`
+	InitialHandshakeTimeoutSeconds int    `json:"initialHandshakeTimeoutSeconds,omitempty" yaml:"Initial-handshake-timeout-seconds",omitempty"`
```

Comment:
In the `InitialHandshakeTimeoutSeconds` property: should the yaml mapping start with lower case?